android/SherpaOnnxJavaDemo/app/src/main/java/com/k2fsa/sherpa/onnx/OnlineRecognizer.java

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022-2023 by zhaoming
```

Comment:
By the way, can you use a symlink to
https://github.com/k2-fsa/sherpa-onnx/blob/master/sherpa-onnx/kotlin-api/OnlineRecognizer.kt
to replace this file and other files related to sherpa-onnx Java APIs?

I think you can invoke directly Kotlin APIs in Android Java.